Posts/2021/04/get-live-servers.md

```diff
@@ -0,0 +1,161 @@
+---
+post_title: Testing The Connection to Computers in the Active Directory
+username: tfl@psp.co.uk
+Catagories: PowerShell
+tags: Active Directory, networking
+Summary: How can I get AD computers check to see they are online?
+---
+
+**Q:** As an administrator, I often have to do a lot of reporting on the servers in my domain.
+Is there a simple way to test the connection to every server in my domain or every server or client host in a specific OU?
+
+**A:**  Of course you can do this with PowerShell! You can use the Active Directory cmdlets and `Test-Connection`, although it is not as simple as one might like.
+
+## Using the `ActiveDirectory` module
+
+Microsoft has developed several modules to help you deploy and manage AD in your organisation or via Azure.
+The `ActiveDirectory` module is one which Microsoft ships with Windows Server (although not installed by default).
+You can also load the Remote Server Administration (RSAT) module for AD on a Windows 10 host.
+The RSAT module allows you to manage the AD using PowerShell from a remote machine.
+For more details on the `ActiveDirectory` module, see the [ActiveDirectory module documentation](https://docs.microsoft.com/powershell/module/addsadministration/().)
+
+You can use the `Get-ADComputer` account to return details about some or all computers within the AD.
+You have several ways to use `Get-ADComputer` to get just the computer accounts you want with any property you need.
+These include using the **-Identity** and **-Filter** parameters.  
```

Comment:
See the PowerShell style guide. When referencing parameters with the dash character, use backticks. But when talking about parameters in a paragraph, use bold without the dash. You should only use the dash form when you are specifically talking about syntax.

```suggestion
These include using the **Identity** and **Filter** parameters.  
```